SwiftGit2.xcworkspace/xcshareddata/SwiftGit2.xcscmblueprint

```diff
@@ -0,0 +1,107 @@
+{
```

Comment:
This file really shouldn't be checked in. Can you remove it and then add `*.xcscmblueprint` to the `.gitignore`.